cmd/explaintest/r/clustered_index.result

```diff
@@ -26,28 +26,28 @@ load stats 's/wout_cluster_index_tbl_3.json';
 load stats 's/wout_cluster_index_tbl_4.json';
 explain select count(*) from with_cluster_index.tbl_0 where col_0 < 5429  ;
 id	estRows	task	access object	operator info
-StreamAgg_17	1.00	root		funcs:count(Column#8)->Column#6
-└─IndexReader_18	1.00	root		index:StreamAgg_9
-  └─StreamAgg_9	1.00	cop[tikv]		funcs:count(1)->Column#8
-    └─IndexRangeScan_16	798.90	cop[tikv]	table:tbl_0, index:idx_3(col_0)	range:[-inf,5429), keep order:false
+StreamAgg_18	1.00	root		funcs:count(Column#8)->Column#6
```

Comment:
can we use `explain format = 'brief' select ...` to avoid the effects from ID?

Reply:
yeah, but there are lots of tests and a huge amout of work need to be done.